### DISCUSSION TITLE
Replace the Hot-Rod?

### DISCUSSION THREAD

**Discussion by SMG241** [blackfyre/flsun.community#17] [Q&A]
Hello All,

I had set a cura print for nozzle temp at 260 degrees.  But, after that, neither the bed nor the nozzle heats up. I reflashed the firmware, but that didn't fix it.   Is there some sort of reset, or do I need to replace the Hot-Rod.  

Richard in AZ

**Comment by blackfyre** [comment]
Hi @SMG241!

Welcome the forum and a warning 1st! PTFE tubes break down when they're in contact with temperatures above 240 °C, burning is the least of the problems, it releases noxious fumes as well.
Also the factory hotend & software setup doesn't allow you to any higher really...

**Reply by kyleg11** [reply]
what is meant by FLsun's "factory hotend & software setup doesn't allow you to any higher really..." Do you mean above 260? If so you are correct on that, but I think this person (like myself) is looking to upgrade (this person because perhaps his hotend heating has failed completely)...do you  have a recommendation for a FLsunn QQ-S pro hotend replacement? in general?

**Reply by pingywon** [reply]
Shop around, but this is what i put on mine.

https://smile.amazon.com/E3D-All-Metal-HotEnd-Full-add/dp/B00X6JOL7A/ref=sr_1_4?crid=28FXJV4P05PQ9&keywords=e3d+v6+hotend&qid=1648657557&s=industrial&sprefix=e3d+%2Cindustrial%2C79&sr=1-4
